### PR TITLE
update guidelines to clarify dependency on GH Copilot for chat

### PR DESCRIPTION
**Pull request by isidorn**
fyi @ntrogh @Matv-stripe

### CODE REVIEW

File: api/extension-guides/language-model.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ Once you have created your AI extension, you can publish your extension to the V
...
 - By publishing to the VS Marketplace, your extension is adhering to the [GitHub Copilot extensibility acceptable development and use policy](https://docs.github.com/en/early-access/copilot/github-copilot-extensibility-platform-partnership-plugin-acceptable-development-and-use-policy).
 - Update the attributes in the `package.json` to make it easy for users to find your extension. Add "AI" to the `categories` field in your `package.json`. If your extension contributes a Chat Participant, add "Chat" as well.
+- If your extension already contributes non Language Model functionality, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). That way your extension users that do not use GitHub Copilot can use your non Language Model functionality without GitHub Copilot being installed. Make sure to have appropraite error handling when accessing language models for this case.
```
Comment (matv-stripe) [inline api/extension-guides/language-model.md:218]:
```suggestion
- If your extension already contributes non Language Model functionality, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). That way your extension users that do not use GitHub Copilot can use your non Language Model functionality without GitHub Copilot being installed. Make sure to have appropriate error handling when accessing language models for this case.
```

**Review by rzhao271 — APPROVED** [review]

File: api/extension-guides/chat.md [hunk 2]
```diff
@@ -338,6 +338,7 @@ Once you have created your AI extension, you can publish your extension to the V
 - By publishing to the VS Marketplace, your extension is adhering to the [GitHub Copilot extensibility acceptable development and use policy](https://docs.github.com/en/early-access/copilot/github-copilot-extensibility-platform-partnership-plugin-acceptable-development-and-use-policy).
 - Update the attributes in the `package.json` to make it easy for users to find your extension. Add "AI" and "Chat" to the `categories` field in your `package.json`.
 - Upload to the Marketplace as described in [Publishing Extension](https://code.visualstudio.com/api/working-with-extensions/publishing-extension).
+- If your extension already contributes non Chat functionality, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). That way your extension users that do not use GitHub Copilot can use your non Chat functionality without GitHub Copilot being installed.
```
Comment (ntrogh) [inline api/extension-guides/chat.md:341]:
```suggestion
- If your extension already contributes functionality other than chat, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). This ensures that extension users that do not use GitHub Copilot can use the non-chat functionality without having to install GitHub Copilot.
```

File: api/extension-guides/language-model.md [hunk 3]
```diff
@@ -215,6 +215,7 @@ Once you have created your AI extension, you can publish your extension to the V
...
 - By publishing to the VS Marketplace, your extension is adhering to the [GitHub Copilot extensibility acceptable development and use policy](https://docs.github.com/en/early-access/copilot/github-copilot-extensibility-platform-partnership-plugin-acceptable-development-and-use-policy).
 - Update the attributes in the `package.json` to make it easy for users to find your extension. Add "AI" to the `categories` field in your `package.json`. If your extension contributes a Chat Participant, add "Chat" as well.
+- If your extension already contributes non Language Model functionality, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). That way your extension users that do not use GitHub Copilot can use your non Language Model functionality without GitHub Copilot being installed. Make sure to have appropriate error handling when accessing language models for this case.
```
Comment (ntrogh) [inline api/extension-guides/language-model.md:218]:
```suggestion
- If your extension already contributes functionality other than using the Language Model API, we recommend that you do not introduce an extension dependency on GitHub Copilot in the [extension manifest](/api/references/extension-manifest). This ensures that extension users that do not use GitHub Copilot can use the non language model functionality without having to install GitHub Copilot. Make sure to have appropriate error handling when accessing language models for this case.
```

File: api/extension-guides/chat.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ The following screenshot shows the different chat concepts in the Visual Studio
 
 ## Develop a chat extension
 
-A chat extension is an extension that has a dependency on the [Copilot Chat extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) and contributes a chat participant. You can find details about how to define `extensionDependencies` in the [extension manifest](/api/references/extension-manifest).
+A chat extension is an extension that contributes a chat participant to the [Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) view.
```
Comment (ntrogh) [inline api/extension-guides/chat.md:72]:
```suggestion
A chat extension is an extension that contributes a chat participant to the [Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) view.
```

**Review by ntrogh — COMMENTED** [review]
Added some text suggestions.

**Comment by isidorn** [comment]
Thanks Nick for the suggestions. I have applied all of them. 
This is ready for review.

**Review by alexr00 — APPROVED** [review]

**Pull request merged**